framework/src/meshgenerators/RenameBlockGenerator.C

```diff
@@ -69,29 +73,29 @@ RenameBlockGenerator::validParams()
 RenameBlockGenerator::RenameBlockGenerator(const InputParameters & parameters)
   : MeshGenerator(parameters), _input(getMesh("input"))
 {
-  if (isParamValid("old_block_id") && isParamValid("old_block_name"))
+  if (isParamSetByUser("old_block_id") && isParamSetByUser("old_block_name"))
```

Comment:
I don't understand the need for this change. Looks to me as if this code could just stay completely unchanged (no default above and continuing to use isParamValid)

Reply:
It failed tests if keep unchanged

Reply:
The isParamSetByUser() is removed same as all the rest below